alf/bin/train_test.py

```diff
@@ -34,17 +36,27 @@ def run_and_stream(cmd, cwd):
         cwd (str): working directory for the process
     """
     logging.info("Running %s", " ".join(cmd))
+
+    # create a logger for sub process outputs
+    # 1. logging all outputs of sub process to sys.stderr to make it traceable when an error
+    #   occurs (ci suppresses stdout output to prevent producing a big log file than 4 MB)
+    # 2. set a simple formatter without prefix for the logger, because a log_prefix
+    #   already exists for sub process log
+    logger = logging.ABSLLogger('')
+    handler = sys_logging.StreamHandler(sys.stderr)
+    handler.setFormatter(sys_logging.Formatter('%(message)s'))
+    logger.addHandler(handler)
+
     process = subprocess.Popen(
         cmd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, cwd=cwd)
 
     while process.poll() is None:
         with io.TextIOWrapper(process.stdout, encoding="utf-8") as text_io:
             for line in text_io:
-                logging.info(line.strip())
+                logger.info(line.strip())
```

Comment:
Now the original stdout goes to stderr, and the original stderr goes to stdout (because stderr=subprocess.STDOUT at line 50). So I am still confused. And why not simply use stdout=subprocess.STDERR at line 50?